### PR TITLE
fix: Issue-290 Adding host prop to set container in portal

### PR DESCRIPTION
**Pull request by yogen9**

### CODE REVIEW

**Comment by yogen9** [comment]
#290

File: src/KBarPortal.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { useKBar } from "./useKBar";
 
 interface Props {
   children: React.ReactNode;
+  host?: HTMLElement;
```
Comment (timc1) [inline src/KBarPortal.tsx:8]: Let's just use the same name as what's passed through to Portal:

```suggestion
  container?: HTMLElement;
```

Comment (yogen9) [reply src/KBarPortal.tsx]: sure, done. 

Let me know if extending KBarPortal Prop with Radix Portal is a good idea. 

Screenshot 2023-02-20 at 11 08 35 PM

Comment (yogen9) [reply src/KBarPortal.tsx]: sure, done.

Let me know if extending KBarPortal Prop with Radix Portal is a good idea.

Screenshot 2023-02-20 at 11 09 47 PM

Comment (timc1) [reply src/KBarPortal.tsx]: PortalProps includes `children` by default yea?

Comment (yogen9) [reply src/KBarPortal.tsx]: Yup. it has default `children` prop.

Comment (timc1) [reply src/KBarPortal.tsx]: so we can just do:

```
export function `KBarPortal(props: PortalProps) {
  // ...
}
```

Comment (yogen9) [reply src/KBarPortal.tsx]: yes we can. should i make the changes accordingly ?

**Review by timc1 — APPROVED** [review]

**Pull request merged**